sagemaker-debugger/build_your_own_container_with_debugger/docker/tf_keras_resnet_byoc.py

```diff
@@ -0,0 +1,110 @@
+"""
+This script is a ResNet training script which uses Tensorflow's Keras interface, and acts as an
+example of how to use SageMaker Debugger if you use either your own custom container to run in SageMaker, or your own script outside SageMaker.
+It has been orchestrated with SageMaker Debugger hooks to allow saving tensors during training.
+These hooks have been instrumented to read from json configuration that SageMaker will put in the training container.
+Configuration provided to the SageMaker python SDK when creating a job will be passed on to the hook.
+This allows you to use the same script with differing configurations across different runs.
+
+If you use an official SageMaker Framework container (i.e. AWS Deep Learning Container), then
+you do not have to orchestrate your script as below. Hooks will automatically be added in those environments. This experience is termed as "Zero Script Change". For more information, please refer to https://github.com/awslabs/sagemaker-debugger/blob/master/docs/sagemaker.md#zero-script-change. An example of the same is provided at https://github.com/awslabs/amazon-sagemaker-examples/sagemaker-debugger/tensorflow2/tensorflow2_zero_code_change.
```

Comment:
Change to "Hooks are automatically added in those environments. This experience is called a _zero script change_. For more information, see..."